translate/src/context/Editor.tsx

```diff
@@ -350,10 +352,12 @@ function updateRichValue(
     }
 
     // Need to let react-dom "fix" the select position before setting it right
```

Comment:
@mathjazz This comment is actually already in the code as an explainer. Now it's just more accurate.

Reply:
Yes, I meant on the backend.